Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2008-10-08
Initial Package Version: 4.26
Upstream Status:         Attempting to notify upstream of the problem
Origin:                  Self
Description:             Fixes the -e (--exclude) parameter to actually do
                         what the documentation says it should do


diff -Naur file-orig/doc/file.man file-4.26/doc/file.man
--- file-orig/doc/file.man	2008-03-07 15:00:07.000000000 +0000
+++ file-4.26/doc/file.man	2008-10-08 13:57:37.000000000 +0000
@@ -192,7 +192,7 @@
 Don't consult magic files.
 .It tar
 Don't examine tar files.
-.It token
+.It tokens
 Don't look for known tokens inside ascii files.
 .It troff
 Don't look for troff sequences inside ascii files.

diff -Naur file-orig/src/file.c file-4.26/src/file.c
--- file-orig/src/file.c	2008-07-03 15:48:18.000000000 +0000
+++ file-4.26/src/file.c	2008-10-08 13:59:14.000000000 +0000
@@ -148,9 +148,11 @@
 		{ "ascii",	MAGIC_NO_CHECK_ASCII },
 		{ "compress",	MAGIC_NO_CHECK_COMPRESS },
 		{ "elf",	MAGIC_NO_CHECK_ELF },
+		{ "fortran",	MAGIC_NO_CHECK_FORTRAN },
 		{ "soft",	MAGIC_NO_CHECK_SOFT },
 		{ "tar",	MAGIC_NO_CHECK_TAR },
 		{ "tokens",	MAGIC_NO_CHECK_TOKENS },
+		{ "troff",	MAGIC_NO_CHECK_TROFF },
 	};
 
 	/* makes islower etc work for other langs */
